app/kubernetes/views/applications/edit/applicationController.js

```diff
@@ -366,6 +367,7 @@ class KubernetesApplicationController {
       expandedNote: false,
       useIngress: false,
       useServerMetrics: this.EndpointProvider.currentEndpoint().Kubernetes.Configuration.UseServerMetrics,
+      publicUrl: this.EndpointProvider.currentEndpoint().PublicURL,
```

Comment:
is PublicURl always available in all cases of local, agent and edge?

Reply:
should be